lib/sourceParams.js

```diff
@@ -162,6 +163,9 @@ module.exports = function (app) {
   app.set('jsCompiledOutput', path.join(appDir, params.staticsRoot, params.js.output))
   app.set('jsBundledOutput', path.join(app.get('jsPath'), params.js.bundler.output))
 
+  // map shared template files
+  app.set('clientViewBundles', params.clientViewBundles)
```

Comment:
Since you're exposing this as an express variable, be sure to add it to the [Express variables exposed by Roosevelt table](https://github.com/rooseveltframework/roosevelt/blob/f0c6765e7ebbbc927d833b5bcc69e4d300bc1c48/README.md#express-variables-exposed-by-roosevelt)

Reply:
Done